src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -68,18 +70,14 @@ package object servlets extends DebugEnhancedLogging {
 
   implicit class RichZipInputStream(val zipInputStream: ZipInputStream) extends AnyVal {
     def unzipPlainEntriesTo(dir: File): Try[Unit] = {
-      def extract(entry: ZipEntry) = {
+      def extract(entry: ZipEntry): Try[Any] = {
```

Comment:
```suggestion
      def extract(entry: ZipEntry): Try[Unit] = {
```

Reply:
Does not compile, the if returns a Try[File], the else a Try[Long]